casper/src/main/resources/Either.rho

```diff
@@ -23,6 +58,12 @@ new Either, rs(`rho:registry:insertSigned:ed25519`), uriOut, rl(`rho:registry:lo
       }
     } |
 
+    contract Either(@"map <-", eitherCh, f, return) = {
+      for (@either <- eitherCh) {
+        Either!("map", either, *f, *return)
+      }
+    } |
+
```

Comment:
please add tests for all these new methods

Reply:
Will do either in this PR or in a follow up :+1: For now, I think you can tell from the implementations and the RevVaultTest.rho passing that those are OK